luminoso_api/client.py

```diff
@@ -125,7 +125,7 @@ def _get_token_auth(username, password, token, token_file, root_url):
             if username is None:
                 username = os.environ['USER']
             if password is None:
-                password = getpass('Password for %s: ' % username)
+                password = getpass(('Password for %s: ' % username).encoding('utf8'))
```

Comment:
It's `.encode()`, not `.encoding()`, I think, and we conventionally spell it `'utf-8'` in other Luminoso code.

This also runs into a bit of a problem on python 3, at least in Linux:

```python
lin@calvin:~$ python3
Python 3.5.2 (default, Sep 10 2016, 08:21:44) 
[GCC 5.4.0 20160609] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from getpass import getpass
>>> getpass('I am a prompt'.encode('utf-8'))
b'I am a prompt'
```

I think `getpass()` just literally prints the prompt argument, whether it's a string or not.

Have you tested this locally?